src/io/flutter/preview/PreviewView.java

```diff
@@ -708,6 +708,13 @@ public void customizeCellRenderer(
       appendSearch(outline.getVariableName(), SimpleTextAttributes.REGULAR_ATTRIBUTES);
     }
 
+    // Render a generic label.
+    if (outline.getKind().equals(FlutterOutlineKind.GENERIC) && outline.getLabel() != null) {
```

Comment:
I think the previous `Widget` label worked pretty well here. I do think that this label will end up being too long.

What about `Widget` _buildListForSessions( ... )` or similar? So the actual method called is part of the suffix text of the node, and is partially elided for long items, either in IntelliJ or on the server.

Reply:
Done in Analysis Server.
So, in the plugin we can continue rendering the whole label as is.

![image](https://user-images.githubusercontent.com/384794/35832110-e16764f6-0a80-11e8-956a-46b09df98015.png)